Dockerfile

```diff
@@ -162,7 +163,8 @@ RUN find . -path '*/coverage/*' -o -path '*/test/*' -o -path '*/.nyc_output/*' \
 		-o -name '*.yml' \
 		-delete \
 	&& find . -type f -path '*/node_modules/sqlite3/deps*' -delete \
-	&& find . -type f -path '*/node_modules/knex/build*' -delete
+	&& find . -type f -path '*/node_modules/knex/build*' -delete \
+	&& rm -f node_modules/sqlite3/node.dtps
```

Comment:
I think `rm` will fail if the file is not found -- might make sense to "catch" this case so it doesn't start failing suddenly.

Reply:
the `-f` makes it ignore non-existent files.
I just added a `-r` too cause it turned out it was a directory.